multi-model-service/docs/code/access.md

```diff
@@ -0,0 +1,271 @@
+# Access & Usage Information
+
+## Basic Information
+
+### Model Server URL
+
+The model server can be accessed [here](https://model-service.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/).
+
+## Authentication
+
+In order to gain access to the model server you will need to sign in with your SSO credentials. Once you have signed in you are able to generate a token by navigating to **`Apps and API Keys`** on the top ribbon and hitting **`Create new Application`**:
+
+![Generation Example](../images/generation-example.png)
+
+**Note:** If you are unable to access the model server via SSO you need to reach out to a platform administrator.
+
+## API Schema
+<!--
+The name of the api, model-service-api, is grabbed from the name field in the ai-catalog.yaml metadata for the api.
+
+TechDocs read this as a URL starting from where the docs are located, so we can use ../ behaviour to navigate the TechDocs to reference other resources/components/apis
+-->
+The API Schema is available [here](../../../../../api/model-service-api/definition).
+
+## Usage Examples
+<!--
+Sourced from: https://model-service.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/examples
+-->
+
+### IBM Granite-8B-Code-Instruct
+
+#### Text Generation
+
+##### Using Curl
+
+```
+curl -X 'POST' \
+    'https://ibm-granite-8b-code-instruct-3scale-apicast-production.<hostname>:<port>/v1/completions' \
+    -H 'accept: application/json' \
+    -H 'Content-Type: application/json' \
+    -H 'Authorization: Bearer ***************************' \
+    -d '{
+    "model": "ibm-granite-8b-code-instruct",
+    "prompt": "San Francisco is a",
+    "max_tokens": 15,
+    "temperature": 0
+}'
+```
+
+##### Python
+
+```python
+import requests
+import urllib3
+import numpy as np
+import json
+
+API_URL = "https://ibm-granite-8b-code-instruct-3scale-apicast-production.<hostname>:<port>"
+API_KEY = "***************************"
+
+input = ["San Francisco is a"]
+
+completion = requests.post(
+    url=API_URL+'/v1/completions',
+    json={
+      "model": "ibm-granite-8b-code-instruct",
+      "prompt": "San Francisco is a",
+      "max_tokens": 15,
+      "temperature": 0
+    },
+    headers={'Authorization': 'Bearer '+API_KEY}
+).json()
+
+print(completion)
+```
+
+##### Python With Langchain
+
+**Note:** Requires `pip install langchain-community`
+
+```python
+from langchain_community.llms import VLLMOpenAI
+
+API_URL = "https://ibm-granite-8b-code-instruct-3scale-apicast-production.<hostname>:<port>"
+API_KEY = "***************************"
+
+llm = VLLMOpenAI(
+    openai_api_key=API_KEY,
+    openai_api_base=API_URL+"/v1",
```

Comment:
```suggestion
    openai_api_base=f"{API_URL}/v1",
```

Reply:
Same as my other reply, these examples were taken right from their docs so I'm not sure if we want to change them